feature/mpls/sr/otg_tests/srte_egress_node_forwarding/egress_node_forwading_test.go

```diff
@@ -0,0 +1,674 @@
+	flowV4.Rate().SetPps(trafficRatePps)
+	flowV4.Duration().FixedPackets().SetPackets(packetsPerFlow)
+
+	ethHdrV4 := flowV4.Packet().Add().Ethernet()
+	ethHdrV4.Src().SetValue(atePort1.MAC)
+	ethHdrV4.Dst().Auto() // MAC of DUT's ingress port
+
+	mplsHdrV4 := flowV4.Packet().Add().Mpls()
+	mplsHdrV4.Label().SetValue(mplsLabelV4)
+	mplsHdrV4.TrafficClass().SetValue(mplsExpBit)
+	mplsHdrV4.BottomOfStack().SetValue(mplsSBit)
+	mplsHdrV4.TimeToLive().SetValue(mplsOuterTTL)
+
+	ipHdrV4 := flowV4.Packet().Add().Ipv4()
+	ipHdrV4.Src().SetValue(ateTrafficSrcIPv4)
+	ipHdrV4.Dst().SetValue(ateTrafficDstIPv4)
+
+	tcpHdrV4 := flowV4.Packet().Add().Tcp()
+	tcpHdrV4.SrcPort().SetValue(10000 + (packetsPerFlow % 50000))
+	tcpHdrV4.DstPort().SetValue(tcpDstPort)
+
+	udpHdrV4 := flowV4.Packet().Add().Udp()
+	udpHdrV4.SrcPort().SetValue(10000 + (packetsPerFlow % 50000))
+	udpHdrV4.DstPort().SetValue(udpDstPort)
+
+	// Traffic Flow 2: Ethernet+MPLS+IPv6+Payload
+
+	t.Logf("Configuring ATE Flow: %s", flowV6Name)
+	flowV6 := otgConfig.Flows().Add().SetName(flowV6Name)
+	flowV6.Metrics().SetEnable(true)
+	flowV6.TxRx().Device().SetTxNames([]string{atePort1.Name + ".IPv6"}).SetRxNames([]string{atePort2.Name + ".IPv6"})
+	flowV6.Size().SetFixed(trafficFrameSize)
+	flowV6.Rate().SetPps(trafficRatePps)
+	flowV6.Duration().FixedPackets().SetPackets(packetsPerFlow)
+
+	ethHdrV6 := flowV6.Packet().Add().Ethernet()
+	ethHdrV6.Src().SetValue(atePort1.MAC)
+	ethHdrV6.Dst().Auto()
+
+	mplsHdrV6 := flowV6.Packet().Add().Mpls()
+	mplsHdrV6.Label().SetValue(mplsLabelV6)
+	mplsHdrV6.TrafficClass().SetValue(mplsExpBit)
+	mplsHdrV6.BottomOfStack().SetValue(mplsSBit)
+	mplsHdrV6.TimeToLive().SetValue(mplsOuterTTL)
+
+	ipHdrV6 := flowV6.Packet().Add().Ipv6()
+	ipHdrV6.Src().SetValue(ateTrafficSrcIPv6)
+	ipHdrV6.Dst().SetValue(ateTrafficDstIPv6)
+
+	tcpHdrV6 := flowV4.Packet().Add().Tcp()
+	tcpHdrV6.SrcPort().SetValue(10000 + (packetsPerFlow % 50000)) // pseudo-random src port > 1024
+	tcpHdrV6.DstPort().SetValue(tcpDstPort)
+
+	udpHdrV6 := flowV6.Packet().Add().Udp()
+	udpHdrV6.SrcPort().SetValue(20000 + (packetsPerFlow % 40000)) // pseudo-random src port > 1024
+	udpHdrV6.DstPort().SetValue(udpDstPort)
+
+	return otgConfig
+}
+
+func enableCapture(t *testing.T, config gosnappi.Config, port string) {
+	config.Captures().Clear()
+	t.Log("Enabling capture on ", port)
+	config.Captures().Add().SetName(port).SetPortNames([]string{port}).SetFormat(gosnappi.CaptureFormat.PCAP)
+
+}
+
+func startCapture(t *testing.T, otg *otg.OTG) gosnappi.ControlState {
+	t.Helper()
+	cs := gosnappi.NewControlState()
+	cs.Port().Capture().SetState(gosnappi.StatePortCaptureState.START)
+	otg.SetControlState(t, cs)
+
+	return cs
+}
+
+func stopCapture(t *testing.T, otg *otg.OTG, cs gosnappi.ControlState) {
+	t.Helper()
+	cs.Port().Capture().SetState(gosnappi.StatePortCaptureState.STOP)
+	otg.SetControlState(t, cs)
+}
+
+func processCapture(t *testing.T, otg *otg.OTG, port string) string {
```

Comment:
https://google.github.io/styleguide/go/decisions#must-functions

Function should probably be renamed to `mustProcessCapture`
```suggestion
func mustProcessCapture(t *testing.T, otg *otg.OTG, port string) string {
```